src/redis_cmd.cc

```diff
@@ -4473,6 +4524,8 @@ CommandAttributes redisCommandTable[] = {
     ADD_CMD("clusterx", -2, "cluster no-script", 0, 0, 0, CommandClusterX),
 
     ADD_CMD("eval", -3, "exclusive write no-script", 0, 0, 0, CommandEval),
+    ADD_CMD("evalsha", -3, "exclusive write no-script", 0, 0, 0, CommandEvalSHA),
+    ADD_CMD("script", -2, "exclusive read-only no-script", 0, 0, 0, CommandScript),
```

Comment:
```suggestion
    ADD_CMD("script", -2, "exclusive no-script", 0, 0, 0, CommandScript),
```
remove `read-only` attribute ?

Reply:
we allow executing `script exists` command on slave, so it should perform as `read-only`

Reply:
if no `read-only` attribute, we don't consider it as write command, right?

Reply:
actually, some subcommands weren't, but it's ok that we only allow the `exists` subcommand when the role was a slave.